src/sound.c

```diff
@@ -15,36 +15,36 @@
 
 #if defined(FEAT_SOUND) || defined(PROTO)
 
-static long	    sound_id = 0;
+static long sound_id = 0;
 
 // soundcb_T is typedef'ed in proto/sound.pro
 
 struct soundcb_S {
-    callback_T	snd_callback;
-#ifdef MSWIN
-    MCIDEVICEID	snd_device_id;
-    long	snd_id;
-#endif
-    soundcb_T	*snd_next;
+    callback_T snd_callback;
+# ifdef MSWIN
+    MCIDEVICEID snd_device_id;
+    long snd_id;
+# endif
+    soundcb_T *snd_next;
 };
 
-static soundcb_T    *first_callback = NULL;
+static soundcb_T *first_callback = NULL;
 
 /*
  * Return TRUE when a sound callback has been created, it may be invoked when
  * the sound finishes playing.  Also see has_sound_callback_in_queue().
  */
-    int
+int
```

Comment:
The return type should be indented by one 'shiftwidth'.

Reply:
This might be the on rule that could cause a problem. I'll see what I can do.